frontend/src/v5/validation/containerAndFederationSchemes/settingsSchemes.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import * as Yup from 'yup';
+import { formatMessage } from '@/v5/services/intl';
+import { EMPTY_VIEW } from '@/v5/store/store.helpers';
+import { numberField } from './validators';
+
+export const BaseSettingsSchema = Yup.object().shape({
```

Comment:
I would change this to `SettingsSchema`  unless there is a good reason to keep this naming

Reply:
Most of the schemes were just moved to new files, but their names was kept as it was before. I have changes this to be `SettingsSchema`